DXMainClient/DXGUI/GameClass.cs

```diff
@@ -379,8 +389,18 @@ public static void SetGraphicsMode(WindowManager wm)
 
             if (borderlessWindowedClient)
             {
-                graphics.IsFullScreen = true;
-                graphics.ApplyChanges();
+                // Note: on full screen mode, the client resolution must exactly match the desktop resolution. Otherwise buttons outside of client resolution are unclickable.
+                ScreenResolution clientResolution = (windowWidth, windowHeight);
+                if (ScreenResolution.DesktopResolution == clientResolution)
+                {
+                    Logger.Log($"Entering full screen mode with resolution {ScreenResolution.DesktopResolution}.");
+                    graphics.IsFullScreen = true;
+                    graphics.ApplyChanges();
+                }
+                else
+                {
+                    Logger.Log($"Not entering full screen mode due to resolution mismatch. Desktop: {ScreenResolution.DesktopResolution}, Client: {clientResolution}.");
```

Comment:
```suggestion
                    Logger.Log($"Not entering borderless full screen mode due to resolution mismatch. Desktop: {ScreenResolution.DesktopResolution}, Client: {clientResolution}.");
```

Tho not sure if the `borderless full screen` is the correct term?

Reply:
I doubt. Even if `graphics.IsFullScreen` does not hold, the client is still in a borderless full screen mode. (the XNA build acts like this)

https://docs.monogame.net/api/Microsoft.Xna.Framework.GraphicsDeviceManager.html#Microsoft_Xna_Framework_GraphicsDeviceManager_IsFullScreen

In the documentation above, the full screen mode is just called fullscreen mode. Should be different from borderless window.

Reply:
I meant "borderless window" vs "borderless full screen".

Reply:
the latter. 

"wm.SetBorderlessMode(borderlessWindowedClient)" sets borderless window, which is not the same with IsFullScreen here.